go.mod

```diff
@@ -30,7 +30,8 @@ require (
 replace (
 	// ww/goautoneg repo does not exist on butbucket anymore, hence we
 	// need to use a mirror. It can be removed when we update to newer libary-go
-	bitbucket.org/ww/goautoneg => github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
+	// bitbucket.org/ww/goautoneg => github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
```

Comment:
@pedjak double checking if its ok to remove

Reply:
I think it is.. otherwise, you would not be able to do perform `go mod vendor`